modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx

```diff
@@ -154,6 +157,36 @@ export default function ActionBuilder({
 		return fields;
 	}, [currentObjectDefinitionFields]);
 
+	useEffect(() => {
+		if (values.objectActionTriggerKey === 'onAfterDelete') {
+			newObjectActionExecutors.map((action) => {
```

Comment:
@alinedoleron You need to cover the scenario when the user has select "On After Add" for the trigger (for example) and "Update an Object Entry" for the action, but afterwards they select "On After Delete" for trigger. I believe we should automatically "clean" the action select and return to the state "Choose an Action".